libs/watt/src/lib/components/paginator/watt-paginator.component.ts

```diff
@@ -60,6 +60,7 @@ import { map, startWith } from 'rxjs';
       [pageIndex]="pageIndex()"
       [showFirstLastButtons]="true"
       [attr.aria-label]="description()"
+      role="navigation"
```

Comment:
question: the element has already a `role="group"`. Can we use that instead of assigning it a new role?

![image](https://github.com/user-attachments/assets/6b8bedd5-455b-4cf1-9314-75c907a40ae6)

Reply:
But group is not all that unique? Maybe we find more than one group?

Reply:
Looking at the assertion

```ts
expect(screen.getByRole('navigation', { name: /Select page/i })).toBeInTheDocument();
```

doesn't the `{ name: /Select page/i }` part ensure that the right element will be found?

Reply:
Think you are right, will remove the added role :-)